deploy/helm/sumologic/values.yaml

```diff
@@ -4019,7 +4023,7 @@ metadata:
           - file_storage
           # - sumologic
         pipelines:
-          logs:
+          logs/fluent-bit/containers:
```

Comment:
And perhaps (but I'm not sure) change this to `logs/fluent/containers` 🤔 Or something similar

Reply:
If we're going to name these for input protocol, then that would be the right name, yes. Seems good to me.